<!DOCTYPE html>
<html lang='en'>
<head>
<title>0005-Add-missing-rule-for-gropdf.patch - rpms/groff.git - groff</title>
<meta name='generator' content='cgit v1.1'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://src.fedoraproject.org/cgit/rpms/groff.git/atom/0005-Add-missing-rule-for-gropdf.patch?h=master' type='application/atom+xml'/>
<link rel='vcs-git' href='https://src.fedoraproject.org/git/rpms/groff.git' title='rpms/groff.git Git repository'/>
</head>
<body>
<script src="https://apps.fedoraproject.org/fedmenu/js/jquery-1.11.2.min.js"></script>
<script src="https://apps.fedoraproject.org/fedmenu/js/fedmenu.js"></script>

<script>
    var base = 'https://apps.fedoraproject.org/';
    var tokens = window.location.href.split('/');
  var repo = null;
  if (tokens.length > 5 && tokens[4] == 'rpms' && tokens[5] != '') {
    repo = tokens[5].split('.')[0];
  }
  fedmenu({
      'url': base + 'js/data.js',
      'mimeType': 'application/javascript',
      'position': 'bottom-right',
      'package': repo,
  });
</script>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='rpms/groff.git' href='/cgit/rpms/groff.git/'>rpms/groff.git</a></td><td class='form'><form method='get'>
<select name='h' onchange='this.form.submit();'>
<option value='f10'>f10</option>
<option value='f11'>f11</option>
<option value='f12'>f12</option>
<option value='f13'>f13</option>
<option value='f14'>f14</option>
<option value='f15'>f15</option>
<option value='f16'>f16</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='f22'>f22</option>
<option value='f23'>f23</option>
<option value='f24'>f24</option>
<option value='f25'>f25</option>
<option value='f26'>f26</option>
<option value='f27'>f27</option>
<option value='f28'>f28</option>
<option value='f7'>f7</option>
<option value='f8'>f8</option>
<option value='f9'>f9</option>
<option value='master' selected='selected'>master</option>
</select> <input type='submit' value='switch'/></form></td></tr>
<tr><td class='sub'>groff</td><td class='sub right'>Jesse Keating</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/rpms/groff.git/'>summary</a><a href='/cgit/rpms/groff.git/refs/'>refs</a><a href='/cgit/rpms/groff.git/log/0005-Add-missing-rule-for-gropdf.patch'>log</a><a class='active' href='/cgit/rpms/groff.git/tree/0005-Add-missing-rule-for-gropdf.patch'>tree</a><a href='/cgit/rpms/groff.git/commit/0005-Add-missing-rule-for-gropdf.patch'>commit</a><a href='/cgit/rpms/groff.git/diff/0005-Add-missing-rule-for-gropdf.patch'>diff</a><a href='/cgit/rpms/groff.git/stats/0005-Add-missing-rule-for-gropdf.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/rpms/groff.git/log/0005-Add-missing-rule-for-gropdf.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/rpms/groff.git/tree/'>root</a>/<a href='/cgit/rpms/groff.git/tree/0005-Add-missing-rule-for-gropdf.patch'>0005-Add-missing-rule-for-gropdf.patch</a></div><div class='content'>blob: 2ab9eed07a0f54e5aa4125b9febfa618e8c9c359 (<a href='/cgit/rpms/groff.git/plain/0005-Add-missing-rule-for-gropdf.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
</pre></td>
<td class='lines'><pre><code>From 65c9f2f5152b3eebba8494cbead00d79d1dcbb1c Mon Sep 17 00:00:00 2001
From: Jan Chaloupka &lt;jchaloup&#64;redhat.com&gt;
Date: Wed, 12 Nov 2014 08:23:00 +0100
Subject: [PATCH] Add missing rule for gropdf

<span class="hl kwb">---</span>
 Makefile.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makefile.in b/Makefile.in
index 4b5a5c6..7ae9ac2 100644
<span class="hl kwb">--- a/Makefile.in</span>
<span class="hl kwa">+++ b/Makefile.in</span>
&#64;&#64; -896,6 +896,8 &#64;&#64; $(GNULIBDIRS): FORCE
 	  $(MAKE) ACLOCAL=: AUTOCONF=: AUTOHEADER=: AUTOMAKE=: $(do) ;; \
 	esac
 
<span class="hl kwa">+$(SHPROGDIRS): $(PROGDEPDIRS)</span>
<span class="hl kwa">+</span>
 $(OTHERDIRS): $(PROGDEPDIRS) $(CCPROGDIRS) $(CPROGDIRS) $(SHPROGDIRS)
 
 $(INCDIRS) $(PROGDEPDIRS) $(SHPROGDIRS) $(OTHERDIRS): FORCE
<span class="hl kwb">-- </span>
1.9.3

</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated by <a href='https://git.zx2c4.com/cgit/about/'>cgit v1.1</a> at 2018-03-03 16:33:45 +0000</div>
</div> <!-- id=cgit -->
</body>
</html>
